### ISSUE TITLE
[Feature]: Accessibility - keyboard navigation

### ISSUE THREAD

**Issue by matthieu-crouzet** [RawZ06/ngx-monaco-tree#56]
The component is currently not accessible as we cannot navigate inside with the keyboard.

## Proposal

Tabulation: to jump in and out of the tree
Arrow up/down: to navigate inside a folder
Arrow left/right: to collapse/expand a folder

**Comment by RawZ06** [comment]
Indeed, I initially didn’t plan for the package to be actually used; I had focused on the essentials. I admit I often overlook accessibility, just like a good back-end developer.
I really like the shortcuts you suggested since VSCode has the same ones!
I approve!
